serverless/about-serverless.adoc

```diff
@@ -0,0 +1,78 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="about-serverless"]
+= About {ServerlessProductName}
+:context: about-serverless
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+{ServerlessProductName} enables developers to deploy and manage cloud native applications that can run anywhere, without provisioning or managing servers.
+These routine tasks are abstracted away by the platform, allowing developers to push code to production much faster than in traditional models.
+
+{ServerlessProductName} is based on the open source link:https://knative.dev/[Knative] project, which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
+
+Developers can use a single platform for hosting their microservices, legacy, and serverless applications, and can use the provided Kubernetes native APIs, as well as familiar languages and frameworks, to deploy applications and stateless container workloads.
+This improves developer productivity by removing the need for complex infrastructure set up before apps can be deployed.
```

Comment:
Should this say "the need for complex infrastructure to be set up before..." ?

Or maybe "the need for a complex infrastructure setup before..." ?